definitions/elasticsearch-filebeat-kibana/component.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elasticsearch-filebeat-kibana",
+  "generator": "static",
+  "type": "static",
+  "path": "./manifests",
+  "subcomponents": [
+    {
+      "name": "elasticsearch",
+      "generator": "helm",
```

Comment:
Lets get rid of generator. Been meaning to remove this param from the other definitions for a while